### ISSUE TITLE
The compiler option "strict" should be enabled to reduce type errors

### ISSUE THREAD

**Issue by isRyven** [microsoft/vscode-edge-devtools#1162]
**Environment (please complete the following information):**
 - OS (*e.g.,* Windows 10): Kubuntu 22.04 LTS x86_64 
 - Extension version (*e.g.,* 1.2.3): 2.1.0

**Describe the bug:**

I am not entirely sure why this happens, but it seems like some extension's internal linter rules get bleeded outside into my project, and i am ending up seeing the error like this on all of my tsconfig files: 
```sh
The compiler option "strict" should be enabled to reduce type errors
```
![image](https://user-images.githubusercontent.com/14905030/186479191-cb45bdfe-b5a0-48c2-9be8-df9af031834c.png)

The annotation suggests the error comes from this extension. After disabling the extension the error goes away.

**Repro steps:**

I don't have any specific steps to reproduce, i have a monorepo project with base tsconfig files, and couple of packages, nothing outstanding really, which makes it even more confusing.  However i have a lockfile to share, perhaps some package causing this https://gist.github.com/isRyven/e84f313b36f21126b6a73627efde3b67

**Expected behavior:**

**Additional context:**

**Comment by codepo8** [comment]
I don't think that this means your configuration gets changed. This is just a message that this is a best practice to do. We have a new version coming up that allows you to filter these out.

**Comment by croban** [comment]
how can be ignored? because it's really annoying?

**Comment by croban** [comment]
Turning off Webhint within "Extension settings" solves this issue, but it turns off other hints for accessibility, compatibility, security etc.
